oneview_redfish_toolkit/blueprints/session.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Python libs
+import logging
+
+# 3rd party libs
+from flask import abort
+from flask import Blueprint
+from flask import request
+from flask import Response
+from flask_api import status
+from hpOneView.exceptions import HPOneViewException
+from hpOneView.oneview_client import OneViewClient
+
+# own libs
+from oneview_redfish_toolkit.api.errors import OneViewRedfishError
+from oneview_redfish_toolkit.api.session import Session
+from oneview_redfish_toolkit import util
+
+
+session = Blueprint('session', __name__)
+
+
+@session.route('/redfish/v1/SessionService/Sessions', methods=["POST"])
+def post_session():
+    """Posts Session
+
+        The response to the POST request to create a session includes:
+
+            - An X-Auth-Token header that contains a session auth token that
+            the client can use an subsequent requests.
+            - A Location header that contains a link to the newly created
+            session resource.
+            - The JSON response body that contains a full representation of
+            the newly created session object.
+
+        Exception:
+            HPOneViewException: Invalid username or password.
+            return abort(400)
+
+            OneViewRedfishError: When occur a credential key mapping error.
+            return abort(400)
+
+            Exception: Unexpected error.
+            return abort(500)
+    """
+
+    try:
+        try:
+            body = request.get_json()
+            username = body["UserName"]
+            password = body["Password"]
+        except Exception:
+            raise OneViewRedfishError(
+                {"errorCode": "INVALID_INFORMATION",
+                 "message": "Invalid JSON key"})
```

Comment:
This message is going to be sent back to user, I would not understand from this message that I created a wrong post form. Please try to create a message that tell to user what happened so he can figure out how to fix it.

Reply:
Ok, fixed.